.ci/openjdk14-excluded.files

```diff
@@ -0,0 +1,514 @@
+<!-- This file generated by script at -->
+<!--     https://github.com/checkstyle/checkstyle/issues/8901#issuecomment-712939037 -->
+
+<!-- Excluded due to exception; issues opened -->
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8945 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="/NestedTypes\.java$"/>
```

Comment:
Is there a reason these files do not have any reference to an exact or matching folder path?

By specifying a file only, it means we will exclude any file matching that name, especially if there are 2 or more.

Furthermore why do some have `/` and others don't? For Windows compatibility folder paths should be `[\\/]`

Reply:
https://github.com/checkstyle/checkstyle/issues/8901#issuecomment-712939037

It was easier to catch files and generate this file.
Change to match different files is very low. We can improve this late on ones it become proven way for jdk validation.

Reply:
I understand this file is generated, but I am seeing anything that says why folders were removed. If this is to be done later ok, but we could be missing overlapping files.

Reply:
>Is there a reason these files do not have any reference to an exact or matching folder path?

A good question. There were quite a few duplicates of identical/ nearly identical files with the same name that kept failing with NPE, so in order to generate this list of 
 `BeforeExecutionExclusionFileFilter` modules in a reasonable amount of time (it still took over three days),  I had the script eliminate the path. 

Also, since I semi-manually verified non-compilable files, this allowed me to get it done in less time. Additionally, I have had issues line-wrapping XML reliably, so another reason to omit the paths was to keep lines under 100 chars.

>By specifying a file only, it means we will exclude any file matching that name, especially if there are 2 or more in different folders.

I felt that losing some granularity in the `BeforeExecutionExclusionFileFilter` modules was a fair trade to be able to increase our testing on openjdk14 from around 2,000 files to more than 40,000.  In retrospect, I should have discussed this in issue and got feedback before proceeding. 

>Furthermore why do some have / and others don't?
That's a good question; I didn't catch that.  The file names that are not preceded with '/' were the ones that I added manually that we were already aware of issues with.  I can fix that.

>For Windows compatibility folder paths should be `[\\/]`

We only have a Linux build on Semaphore, so I hadn't considered Windows compatibility, honestly.  I can fix this so that windows users can run locally.

Reply:
Please fix windows compatibility.
Another issue can be postponed.

Reply:
Yea, please fix Windows directories so it can be run by those users locally.

I am ok to skip adding any type of folder paths.

Reply:
To be clear, I need to change all the preceding `/` to `\\/`, right?

Reply:
you have single , so need to change single.
example of how it will looks like:
https://github.com/checkstyle/checkstyle/blob/3fc9dd98ad6ed963e2cfcd2e630f7031ec5b0c1e/config/suppressions.xml#L67

Reply:
Fixed.  I've also removed to escapes from the `\.` in file names to be consistent with your example.